mobify/react-styleguidist
# Merge 4.1.0 into our fork

`react-styleguidist` 4.x made a bunch of code refactoring changes as well as a big move to support Webpack 2. We also were running into issues with our fork working properly with Node 4/npm 2.x.

## Linked PRs
- https://github.com/mobify/progressive-web-sdk/pull/349

## Changes
- Everything

## How to test-drive this PR
- `npm install`
- `npm link`
- cd into `progressive-web-sdk` and run `npm link react-styleguidist`
- Inside `progressive-web-sdk`, run `npm run docs:dev`
- open `http://localhost:9000` (hot module reloading will fail to receive the JSON blob since we are running the application from another port)
- Verify stuff still works
